src/librustc_target/abi/mod.rs

```diff
@@ -969,10 +964,6 @@ impl<'a, Ty> TyLayout<'a, Ty> {
     where Ty: TyLayoutMethods<'a, C>, C: LayoutOf<Ty = Ty> {
         Ty::pointee_info_at(self, cx, offset, param_env)
     }
-    pub fn is_freeze<C>(self, cx: &C, param_env: Ty::ParamEnv) -> bool
```

Comment:
Huh, was this actually not needed? Only `pointee_info_at`?
Or was my original plan to implement `pointee_info_at` in `rustc_target` instead?
I guess I had forgotten about the special-cases in it? A bit surprising, still, though.

Reply:
@eddyb  You did suggest [this](https://github.com/rust-lang/rust/pull/57150#issuecomment-472364882) first, and that's what i tried to do, but then you said [this](https://github.com/rust-lang/rust/pull/60237#discussion_r280834440)

Reply:
I was referring to the `param_env` argument, not `is_freeze`.
But I think in your first link, I did forget that you can't implement `pointee_info_at` in `rustc_target` *anyway*.